source/contracts/trading/Orders.sol

```diff
@@ -213,9 +217,9 @@ contract Orders is DelegationTarget, IOrders {
         return true;
     }
 
-    function fillOrderLog(bytes32 _orderId, address _filler, uint256 _creatorSharesFilled, uint256 _creatorTokensFilled, uint256 _fillerSharesFilled, uint256 _fillerTokensFilled, uint256 _tradeGroupId) public returns (bool) {
+    function fillOrderLog(bytes32 _orderId, address _filler, uint256 _creatorSharesFilled, uint256 _creatorTokensFilled, uint256 _fillerSharesFilled, uint256 _fillerTokensFilled, uint256 _settlementFees, uint256 _tradeGroupId) public returns (bool) {
         Order.Data storage _order = orders[_orderId];
-        FillOrder(_order.market, _order.outcome, _order.tradeType, _orderId, _order.price, _order.creator, _filler, _creatorSharesFilled, _creatorTokensFilled, _fillerSharesFilled, _fillerTokensFilled, _tradeGroupId);
+        Augur(controller.lookup("Augur")).logOrderFilled(_order.market.getUniverse(), _order.market.getShareToken(_order.outcome), _order.creator, _filler, _order.price, _creatorSharesFilled, _creatorTokensFilled, _fillerSharesFilled, _fillerTokensFilled, _settlementFees, _tradeGroupId);
```

Comment:
I would really like to get these two log methods out of Orders.  @tinybike can we remove most of the order details off of these?  The Augur Node should already have the order details in its DB from when the order was created, so supplying them here is redundant and a waste of gas.  It also would allow us to get these logs out of Orders contract and directly into Augur.sol to align with the pattern followed everywhere else.

Reply:
Good point!  We can get rid of `creator` and `price` here, since those are both attached to the order.  However, I think we need to log the `orderId` -- this isn't presently included in `OrderFilled` (my mistake!) but we need it in order to look the order up in the augur-node DB.

I think we need all 4 of the shares/tokens filled fields and the settlement fees field included, since those will be specific to the specific fill (not attached to the order).

Reply:
I believe by dropping creator/price we can pull this log out into the proper location as all other information is available in that location.  The same should be true for the other log.  It definitely seems like all order related logs should include the order ID.

Reply:
What do you mean re: pull the log into the proper location?  It looks like the `OrderFilled` log is already on the Augur.sol contract.

Reply:
The log originates from `FillOrder.sol`, but it routes through `Orders.sol` only because it needs to pickup price/creator along the way.  Without needing those two pieces of information, `FillOrder.sol` can directly call `logOrderFilled` without having to route through an intermediate contract.

Reply:
Ah I see what you mean, yes I agree that would be good.